spec/features/editing_a_cma_case_spec.rb

```diff
@@ -275,7 +275,7 @@
         {
           "content_id" => "77f2d40e-3853-451f-9ca3-a747e8402e34",
           "url" => "https://assets.digital.cabinet-office.gov.uk/media/513a0efbed915d425e000002/asylum-support-image.jpg",
-          "content_type" => "application/jpeg",
+          "content_type" => "application/jpg",
```

Comment:
yes, though that was taken from the whitelist in Whitehall... should we let .jpeg though? I guess it makes sense to?